front/app/containers/Admin/projects/project/analysis/InputsList/InputListItem.tsx

```diff
@@ -60,10 +62,7 @@ const InputListItem = memo(({ input, onSelect, selected }: Props) => {
         >
           {!title_multiloc ||
             (isEmpty(title_multiloc) && author && (
-              <Box display="flex" alignItems="center">
-                <Avatar userId={author.data.id} size={24} />
-                <Text m="0px">{getFullName(author.data)}</Text>
-              </Box>
+              <Text m="0px">{getFullName(author.data)}</Text>
```

Comment:
@IvaKop After starting to add a simple avatar, I realized that there is quite some logic to duplicate from the existing Avatar component, and it didn't feel right or worth it to do so. E.g. to have the same style, to fallback to the same image when there is no avatar, to recognize moderators (which could also be handy here).

Instead, I propose we simply don't show the avatar in the list. This is only even relevant for the case where there is no title (mostly surveys), and that's also the case where we can generally expect single inputs per user. So the avatar doesn't add that much in terms of scanning content faster.

Reply:
Agreed!

Reply:
@IvaKop So is that an approval? :)